Source/carthage/Build.swift

```diff
@@ -213,32 +213,10 @@ public func buildableSDKs(sdks: [SDK], _ scheme: String, _ configuration: String
 			return SignalProducer(values: sdks).map { ($0, identities) }
 		}
 		.flatMap(.Concat) { (sdk: SDK, signingIdentities: [CodeSigningIdentity]) -> SignalProducer<SDK, CarthageError> in
```

Comment:
Agreed but as I'm very very bad with ReactiveCocoa, I don'T really know what it should return. ;(